drivers/amlogic/amports/amvideocap.c

```diff
@@ -64,10 +64,8 @@
 #define MODULE_NAME "amvideocap"
 #define DEVICE_NAME "amvideocap"
 
-#define CAP_WIDTH_MAX      1920
-#define CAP_HEIGHT_MAX     1080
-
-#define BUF_SIZE_MAX      (0x1fffff) /* 1920 * 1088 * 4 */
```

Comment:
Are there any playback regressions / implications of changing these values?

Reply:
**Playback has been unaffected with the 1080p titles I have.  No 4K on my setup though.**

When I was trying to debug the grabber problem and looking through the kernel side of it I was baffled how the commented `1920 * 1088 * 4` was meant to come to 0x1FFFFF, even if it was actually based on `1080` and not `1088`.

I've seen where that `BUF_SIZE_MAX` changed to 0x1FFFFF from 0x800000 in both CoreELEC and OSMC commits (one that claimed it would "Change BUF_SIZE_MAX to 0x600000".  So when the working driver ditched it altogether I was kind of relieved! **I've also not found any code that uses that macro**.

Permitting what appears to be a max 4K resolution just seems to be moving with the times.  I haven't stepped though it but I don't imagine it would end well if a 4K frame was grabbed with a 1080p maximum.  Best case the lights are based on the top-left 25% of the screen and worst an overflow.  **As it's for the grabber `/dev/amvideocap0` I don't expect it would be bad for video playback, which is a separate `/dev/amvideo` device.**